feedback.md

```diff
@@ -0,0 +1,32 @@
+Hi Kim!
+
+Many thanks for this awesome submission. I went through all (I hope) files and have a few follow-up questions - no coding needed, just conceptual stuff!
+
+Hope that's fine! :)
+
+# Worker:
+
+https://github.com/Burgestrand/kbc19/blob/master/lib/tasks/active_job.rake all good here! Clean and concise!
+
+# Jobs:
+
+https://github.com/Burgestrand/kbc19/blob/master/app/jobs/application_job.rb#L5: we may want to file this as a bug in rails/rails; if I remember correctly connection pool errors during deserialize would lead to discard!
```

Comment:
I'm assuming this is referring to the templated comment in the rails generator? I haven't looked into it, but it makes sense to me that some catch-all error handler during deserialize (and thus including connection pool usage) would, in error, throw a deserialization error no matter the _actual_ error… lot's of error here.